helm/ingress-exporter-chart/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
Available since 1.9 https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/